Source/Core/AudioCommon/Mixer.cpp

```diff
@@ -47,26 +47,27 @@ unsigned int CMixer::MixerFifo::Mix(short* samples, unsigned int numSamples,
   u32 indexR = m_indexR.load();
   u32 indexW = m_indexW.load();
 
-  u32 low_waterwark = m_input_sample_rate * SConfig::GetInstance().iTimingVariance / 1000;
-  low_waterwark = std::min(low_waterwark, MAX_SAMPLES / 2);
-
-  float numLeft = (float)(((indexW - indexR) & INDEX_MASK) / 2);
-  m_numLeftI = (numLeft + m_numLeftI * (CONTROL_AVG - 1)) / CONTROL_AVG;
-  float offset = (m_numLeftI - low_waterwark) * CONTROL_FACTOR;
-  if (offset > MAX_FREQ_SHIFT)
-    offset = MAX_FREQ_SHIFT;
-  if (offset < -MAX_FREQ_SHIFT)
-    offset = -MAX_FREQ_SHIFT;
-
   // render numleft sample pairs to samples[]
   // advance indexR with sample position
   // remember fractional offset
 
   float emulationspeed = SConfig::GetInstance().m_EmulationSpeed;
-  float aid_sample_rate = m_input_sample_rate + offset;
```

Comment:
How do you want to control the utilization of the Mixer's FIFO?
Same as for https://github.com/dolphin-emu/dolphin/pull/5241/commits/7434f016cef8a34a313f41e145e7e1c8820de8c9#diff-cae23b9df9a9c03d8dcd03f8e9c0b7a2R157 -- but as we have two FIFOs now, we have to controll both.